SPECS/ostree/ostree.spec

```diff
@@ -123,16 +129,16 @@ install -vdm 755 %{buildroot}%{_sysconfdir}/ostree/remotes.d
 %{_unitdir}/ostree*.service
 %{_unitdir}/ostree-finalize-staged.path
 %{_libdir}/dracut/modules.d/98ostree/*
-%{_mandir}/man1/ostree-admin*
+%{_mandir}/man*/*.gz
 %{_libdir}/systemd/system-generators/ostree-system-generator
-%{_libdir}/systemd/system-preset/91-ostree.preset
 %exclude %{_sysconfdir}/grub.d/*ostree
 %exclude %{_libexecdir}/libostree/grub2*
 %{_libdir}/ostree/ostree-prepare-root
-%{_sysconfdir}/dracut.conf.d/ostree.conf
 %{_libdir}/ostree/ostree-remount
 %{_libdir}/tmpfiles.d/ostree-tmpfiles.conf
-%{_libexecdir}/libostree/*
+%{_libexecdir}/lib%{name}/*
+%{_libexecdir}/installed-tests/*
```

Comment:
same here, this seems a bit broad, does ostree make a sub folder in /etc/libexec/installed-tests/?

Reply:
Yeah, and there are about 20+ test files going inside it, so it made sense to make a broader folder.

Reply:
I did make it a bit deeper, tested, it worked!